clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml

```diff
@@ -7,44 +7,44 @@ spec:
   lookupPolicy:
     local: true
 ---
-apiVersion: build.openshift.io/v1
-kind: BuildConfig
+apiVersion: ci.openshift.io/v1
+kind: MultiArchBuildConfig
+
 metadata:
-  name: manage-clonerefs-arm64
+  name: manage-clonerefs
```

Comment:
Isn't `managed-clonerefs` the proper name?

Reply:
Yes, but this change won't make any difference. We will build it only once. We need to implement a mechanism to build the image on demand. I guess we can follow the strategy build or add a cron configuration.

Reply:
Maybe something similar to `oc start-build` in a periodic?
Either way before start working on that, I wanted to make sure the build and mirror mechanism works as expected and we don't face any issue using the new image.

Reply:
We can't use the `oc start-build`. This command is not related to the resource.

Reply:
I mean developing something similar to that command.